### PR TITLE
[WIP] V1.0.0

### PR DESCRIPTION
**Pull request by GuillaumeLeclerc**
Track progress for the next release

### CODE REVIEW

File: ffcv/pipeline/graph.py [hunk 1]
```diff
@@ -0,0 +1,483 @@
...
+                fix = self.fieldname_to_fix[source_field]
+                metadata = self.metadata[f'f{fix}']
+
+                operation.accept_field(self.handlers[source_field])
+                operation.accept_globals(metadata, self.memory_read)
+
+                next_state, allocation = operation.declare_state_and_memory(state)
+                state_allocation = operation.declare_shared_memory(state)
+
+                if next_state.device.type != 'cuda' and isinstance(operation,
```
Comment (ashertrockman) [inline ffcv/pipeline/graph.py:328]: This causes an error for me (`next_state.device` is a string); removing `.type` seems like a quick fix.

**Comment by alexmyczko** [comment]
Can't wait for more releases...

**Comment by ashertrockman** [comment]
@andrewilyas @GuillaumeLeclerc Are you still interested in merging #154 into V1.0.0? For what it's worth: while I didn't finish the checklist in the PR, some colleagues and I have been using it successfully for a long time. Let me know if there's anything I could do to make this happen.

**Comment by andrewilyas** [comment]
Hi @ashertrockman ! My thought was to merge v1.0 -> master ASAP and update on PyPI, and then merge in this and grayscale support (#226) into a v1.1, does that sound good?

BTW, are there tests for the new augmentations?

**Comment by ashertrockman** [comment]
Yeah, that sounds good to me. There are tests here: https://github.com/libffcv/ffcv/blob/0fa3d7c8a041fec7348025afafe6e4f6fff60f33/tests/test_rand_aug.py. My goal was to very closely replicate the RandAugment implementation in TorchVision, so most of the tests use it as a baseline.

**Comment by andrewilyas** [comment]
Amazing! This should be super easy to merge into v1.1 then. I'm planning on uploading v1.0 to PyPI tonight!

**Pull request merged**